pages/linux/as.md

```diff
@@ -0,0 +1,20 @@
+# as
+
+> Portable GNU assembler.
+> Primarily intended to assemble output from `gcc` to be used by `ld`.
+
+- Assemble a file, writing the output to a.out:
+
+`as {{file.s}}`
+
+- Assemble the output to a given file:
+
+`as {{file.s}} -o {{out.o}}`
+
+- Skip whitespace and comment preprocessing, leading to faster output. (Should only be used for trusted compilers):
```

Comment:
`faster output`? Does this mean it runs faster (i.e. faster operation), or that the output binary is more optimised?

Reply:
faster operation. Does it sound confusing ? I thought it was clear enough :thinking:

Reply:
Yeah, it does a bit

Reply:
I think it's pretty clear actually. :thinking:

Reply:
@sbrl, any thoughts on what this could be changed to in order to make it clearer? 👍

Reply:
Perhaps `output that operates faster` would work better, @pxgamer?

Reply:
Maybe I should just remove that bit altogether. It is just extra information which will anyways becomes apparent once the user runs it.

Reply:
Erm that would imply that the output is generated faster, which comflicts with your earlier statement of

>  operation. Does it sound confusing ? I thought it was clear enough thinking

Reply:
You said - `Does this mean it runs faster (i.e. faster operation)`. I said yes. This is implying the same. Output is generated faster means it runs faster.

Reply:
I'm confused. Nevermind then :P

Reply:
No worries. I will update to the new description :)